dashboard/app/models/pd/application/teacher1819_application.rb

```diff
@@ -788,15 +794,25 @@ def yes_no_response_to_yes_no_score(response)
 
     def able_attend_multiple_to_yes_no_score(response)
       response = response.join
-      if response.start_with?(TEXT_FIELDS[:no_explain])
+      if response.start_with?(TEXT_FIELDS[:no_explain]) || response == NO
         NO
-      elsif response && !response.include?(TEXT_FIELDS[:no_explain])
+      elsif (response && !response.include?(TEXT_FIELDS[:no_explain])) || response == YES
         YES
       else
         nil
       end
     end
 
+    def able_attend_single_to_yes_no_score(response)
+      if [TEXT_FIELDS[:able_to_attend_single], YES].include? response
```

Comment:
Why would this be `YES`? Can't we just check if `response === TEXT_FIELDS[:able_to_attend_single]` ?

See [above](https://github.com/code-dot-org/code-dot-org/pull/20484/files#diff-4191bb74227d8bd50c7d3560babf9669R159)